### DISCUSSION TITLE
Help configuring TELEGRAM NOTIFIES

### DISCUSSION THREAD

**Discussion by g0ze21** [swar/Swar-Chia-Plot-Manager#1008] [General]
Hi all.

Please I need help setting the telegram notifications, I have already set.

**notify_telegram: true**
**telegram_token: 1800000000:AAGE3XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX**

But im getting this error starting the ploter.

![image](https://user-images.githubusercontent.com/84890283/120080808-1e33f300-c080-11eb-947a-0fad1b3ae816.png)

**Comment by alexey1312** [comment]
You can find the answer here: [#634](https://github.com/swar/Swar-Chia-Plot-Manager/discussions/634)

**Reply by g0ze21** [reply]
> You can find the answer here: [#634](https://github.com/swar/Swar-Chia-Plot-Manager/discussions/634)

Thankyou man!
